src/ai_hawk/job_manager.py

```diff
@@ -262,7 +264,7 @@ def get_jobs_from_page(self, scroll=False):
                 browser_utils.scroll_slow(self.driver, jobs_container_scrolableElement)
                 browser_utils.scroll_slow(self.driver, jobs_container_scrolableElement, step=300, reverse=True)
 
-            job_element_list = jobs_container.find_elements(By.CSS_SELECTOR, 'div[data-job-id]')
+            job_element_list = jobs_container.find_elements(By.XPATH, ".//li[contains(@class, 'jobs-search-results__list-item') and contains(@class, 'ember-view')]")
```

Comment:
I would say that rely on "ember-view" instead of "data-job-id" more likely to cause problems again so I would revert this change